webserver/views/api/v1/datasets.py

```diff
@@ -217,7 +217,18 @@ def add_recordings(dataset_id):
 
     :resheader Content-Type: *application/json*
     """
-    raise NotImplementedError
+    class_dict = request.get_json()
+    if not class_dict:
+        raise api_exceptions.APIBadRequest("Data must be submitted in JSON format.")
```

Comment:
We should also check the contents of this dictionary before we do something with it. Does it have the required `class_name` and `recordings` fields? Are all of the recordings ids valid UUIDs?
If the class doesn't exist in the dataset (but the dataset does exist), should we automatically create the class, or is it an error?